app/portainer/components/Dashboard/DashboardItem.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import { Meta, Story } from '@storybook/react';
+
+import { DashboardItem } from './DashboardItem';
+
+const meta: Meta = {
+  title: 'Components/DashboardItem',
+  component: DashboardItem,
+};
+export default meta;
+
+interface StoryProps {
+  value: number;
+  icon: string;
+  comment: string;
+  accessibilityLabel: string;
+}
+
+function Template({ value, icon, comment, accessibilityLabel }: StoryProps) {
+  return (
+    <DashboardItem
+      value={value}
+      icon={icon}
+      comment={comment}
+      accessibilityLabel={accessibilityLabel}
+    />
+  );
+}
+
+export const Primary: Story<StoryProps> = Template.bind({});
+Primary.args = {
+  value: 1,
+  icon: 'fa fa-th-list',
+  comment: 'Resource',
+  accessibilityLabel: 'dashboardItem',
+};
```

Comment:
I think we should have a `dashboardItem with link` story too

Reply:
added